js/ui/appDisplay.js

```diff
@@ -2747,6 +2747,7 @@ class HackAppIcon extends AppIcon {
 
         let app = Clubhouse.getClubhouseApp();
         this._activated = false;
+        this._flatpak_link = Gio.File.new_for_path('/var/lib/flatpak/app/com.hack_computer.Clubhouse/current/active');
```

Comment:
You may be able to get this path out of the `app` object rather than hardcoding it?